x-pack/plugins/endpoint/common/types.ts

```diff
@@ -337,7 +337,7 @@ export type ResolverEvent = EndpointEvent | LegacyEndpointEvent;
 /**
  * The PageId type is used for the payload when firing userNavigatedToPage actions
  */
-export type PageId = 'alertsPage' | 'managementPage' | 'policyListPage';
+export type PageId = 'alertsPage' | 'managementPage' | 'policyListPage' | 'policyDetailsPage';
```

Comment:
I don't think you need this - see comment further below

Reply:
removed